lib/card_validation/validity.rb

```diff
@@ -51,6 +51,14 @@ def double
     digits.reverse.each_with_index.map { |x, i| i.even? ? x * 2 : x }
   end
 
+  def empty?
+    number.respond_to?(:empty?) && number.empty?
```

Comment:
Instead of checking if number responds to `empty?`, would it be beneficial to make sure number returns an object you can call `empty?` on?